eth-types/src/l2_types.rs

```diff
@@ -169,6 +175,10 @@ impl TransactionTrace {
             v: self.v,
             r: self.r,
             s: self.s,
+            #[cfg(feature = "kroma")]
+            source_hash: Some(self.source_hash),
```

Comment:
Though I would prefer if the ordering of the `mint`/`source hash` could be standardized (aka `mint` before `source hash`, both after `s`, I do see that the order of other fields is also not standardized. Besides this comment, LGTM

Reply:
I tried to follow the order of our l2 transaction trace, with minimum diff. An example of our l2 transaction trace is as below.
```
{
            "type": 2,
            "nonce": 1,
            "txHash": "0x993256bed459a2ebf390e4b7c1ab915ef8b48637792f6601297d9cc1423cc21d",
            "gas": 100000,
            "gasPrice": "0xb390d1d4",
            "from": "0x3c44cdddb6a900fa2b585dd299e03d12fa4293bc",
            "to": "0x9fe46736679d2d9a65f0992f2272de9f3c7fa6e0",
            "chainId": "0x385",
            "mint": null,
            "value": "0x0",
            "data": "0xa9059cbb00000000000000000000000090f79bf6eb2c4f870365e785982e1f101e93b9060000000000000000000000000000000000000000000000000400000000000000",
            "isCreate": false,
            "sourceHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
            "v": "0x1",
            "r": "0xdaddb750ce136c22a007a3365b3d7c41535f87f95ce54a29b929e287838ac190",
            "s": "0x2ac416e1d81fc770735cf96d2f396acccbcb4fe69de27384b36ea7696a0e98d1",
            "accessList": [
                {
                    "address": "0x9fe46736679d2d9a65f0992f2272de9f3c7fa6e0",
                    "storageKeys": [
                        "0x215be5d23550ceb1beff54fb579a765903ba2ccc85b6f79bcf9bda4e8cb86034"
                    ]
                }
            ]
        },
```